api/apps/api/src/modules/clone/export/adapters/typeorm-export.repository.ts

```diff
@@ -0,0 +1,16 @@
+import { ResourceId } from '@marxan/cloning/domain';
+import { Injectable } from '@nestjs/common';
+
+import { Export } from '../domain';
+import { ExportRepository } from '../application/export-repository.port';
+
+@Injectable()
+export class TypeormExportRepository implements ExportRepository {
+  find(projectId: ResourceId): Promise<Export | undefined> {
```

Comment:
Is this typing for `projectId` correct?

Reply:
@AaronPerezPerez yes, as it is a `resource` (may be a project/scenario/planning area/whatever we want to consider as resource)